draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
Clock skew distributions are skewed more heavily negative than positive, so we should recommend people to set 2 different bounds. This guidance could probably be written as "a small allowance is reasonable, for example a lower bound of X and and an upper bound of Y". 

@ekr Don't have numbers on how frequently clocks change, but probably more interestingly, enforcing a window of 120s (-120 to +120), cut off 0.1% of 0-RTT connections (which would then fallback to 1rtt)

If we were to enforce a window of 10s (-10 to +10), that would additionally cut off another 0.3% 0rtt connections.